h5grove/encoders.py

```diff
@@ -78,6 +78,18 @@ def npy_stream(array: Sequence[Number]) -> Generator[bytes, None, None]:
         yield chunk.tobytes("C")
 
 
+def convert_csv(data: np.ndarray) -> bytes:
```

Comment:
A naming question, why having `*_encode` and `*_convert` functions, I understand the slight difference but is it needed?

Reply:
I asked myself the same question so I think it is not needed :grin: